### PR TITLE
Tidying

### PR DESCRIPTION
**Pull request by lucas-wills**
Fixed the indentation (I hope).

### CODE REVIEW

File: pristine-zen-garden.html [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+                <a href="/200/" class="design-name">Icicle Outback </a>by<a href="http://www.timovirtanen.com/" class="designer-name"> Timo Virtanen</a>
+              </li>
+              <li>
+                <a href="/199/" class="design-name">CSS ZEN ARMY </a>by<a href="http://www.niceguy.com" class="designer-name"> Carl Desmond</a>
+              </li>
+              <li>
+                <a href="/198/" class="design-name">The Original </a>by<a href="#" class="designer-name"> Joachim Shotter</a>
+              </li>
+            </ul>
+              </nav>
```
Comment (anna-marie-vos) [inline pristine-zen-garden.html]: Hi Good job on the clean up.  The only comment I have is to maybe bring the ` </nav>` tag in line with it's buddy `<nav role="navigation">` in line 67

Comment (anna-marie-vos) [inline pristine-zen-garden.html]: Hi Good job on the clean up.  The only comment I have is to maybe bring the `</nav>` tag in line with it's buddy `<nav role="navigation">` in line 67

Comment (lucas-wills) [reply pristine-zen-garden.html]: Wow, that tag is completely out of it! Thanks for letting me know!!

**Comment by anna-marie-vos** [comment]
Hey Lucas, The code looks awesome. I'm gonna go ahead and merge it.

**Pull request merged**